integration/kubernetes/k8s-job.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+}
+
+@test "Run a job to completion" {
+	job_name="job-pi-test"
+	wait_time=20
+	sleep_time=2
+
+	# Create job
+	kubectl apply -f "${pod_config_dir}/job.yaml"
+
+	# Verify job
+	kubectl describe jobs/"$job_name" | grep SuccessfulCreate
```

Comment:
Maybe we can do this more succinctly with a `kubectl wait` or a `kubectl get -o json | jq` ? The `describe | grep` just feels less 'precise'. Having said that, as we are describing exactly only the job we are looking for, maybe its fine ;-)